predbat/Dockerfile.standalone

```diff
@@ -13,12 +13,16 @@ RUN apt-get install -y python3-venv
 RUN apt-get install -y python3-requests
 RUN apt-get install -y python3-yaml
 #RUN apt-get install -y python3-asyncio
-RUN apt-get install -y python3-aiohttp
 RUN apt-get install csh
-RUN apt-get install -y python3-tz
+RUN apt-get install -y python3-aiohttp
+RUN apt-get install -y python3-tz 
+RUN apt-get install -y vim wget zip 
 
-WORKDIR /config
-COPY rootfs /config
+WORKDIR /wkspace
```

Comment:
What's the purpose of the wkspace?

Reply:
this is where all the downloaded zip files and extraction takes place. in order to keep the config directory clean. been using it for a while with no problem